src/app/global/local_app_settings.nim

```diff
@@ -48,7 +48,7 @@ QtObject:
 
   proc languageChanged*(self: LocalAppSettings) {.signal.}
   proc getLanguage*(self: LocalAppSettings): string {.slot.} =
-    self.settings.value(LAS_KEY_LANGUAGE, newQVariant(DEFAULT_LAS_KEY_LANGUAGE)).stringVal
+    self.settings.value(LAS_KEY_LANGUAGE).stringVal
```

Comment:
We don't want to pretend we are running "translated"; English is not a translation, it's the default/fallback